xrmocap/data_structure/body_model/smpl_data.py

```diff
@@ -486,6 +489,7 @@ def load(self, npz_path: str):
         """
         with np.load(npz_path, allow_pickle=True) as npz_file:
             tmp_data_dict = dict(npz_file)
+
```

Comment:
We do not need blank line in such a short function. Add new line only when it's too long to read.